### ISSUE TITLE
unnecessary_pragma for empty catch block inside a macro

### ISSUE THREAD

**Issue by thorsten-wolf-neptune** [abaplint/abaplint#3049]
Having a Macro with an empty try catch and added the `"#EC NO_HANDLER` pseudocomment will give a false positive for the unnecessary_pragma rule:
![image](https://github.com/abaplint/abaplint/assets/83953375/7366d369-1471-4dbe-a70c-83b0843bc1c2)
Here the sample abap code:
```abap
REPORT zfoobar.

define unnecessary_pragma.
    try.
      catch cx_root.                                    "#EC NO_HANDLER
    endtry.
end-of-definition.
```
and the abaplint.json:
```json
{
  "global": {
  },
  "dependencies": [
  ],
  "syntax": {
    "version": "v702"
  },
  "rules": {
    "unnecessary_pragma": true
  }     
}

```

**Comment by larshp** [comment]
@thorsten-wolf-neptune try again, 2.101.36 published

**Comment by thorsten-wolf-neptune** [comment]
Verified it works with the simple example but also in our code base 👍 
thanks for the fast fix ❤️
